src/views/states/list/components/utils.ts

```diff
@@ -2,7 +2,10 @@ import { InterfaceType, NodeNetworkConfigurationInterface } from '@types';
 
 import { FILTERS_TYPES } from '../constants';
 
-export const interfaceFilters = {
+export const INTERFACE_FILTERS: Record<
```

Comment:
I don't think this qualifies as a constant, so I think it should be `interfaceFilters`.